ports/servoshell/desktop/headed_window.rs

```diff
@@ -4,6 +4,8 @@
 
 //! A winit window implementation.
 
+#[deny(clippy::panic)]
```

Comment:
Doesn't it need to be `#!` to apply to the whole file?

Reply:
You're right. I've fixed this as well as some new errors raised by the rule.